### PR TITLE
ADD MOVEGPT token

### PR DESCRIPTION
**Pull request by ken-movegpt**
We are new launchpad in aptos, please listing my MGPT  token
https://www.coingecko.com/en/coins/movegpt
https://movegpt.io/

### CODE REVIEW

File: defi/src/protocols/data3.ts [hunk 1]
```diff
@@ -45471,6 +45471,29 @@ const data3: Protocol[] = [
...
+      "THE MOST TRUSTED.",
+    chain: "Aptos",
+    logo: `${baseIconsUrl}/movegpt.png`,
+    audits: "2",
+    audit_note: null,
+    gecko_id: "movegpt",
+    cmcId: null,
+    category: "Dexes",
+    chains: ["Aptos"],
+    module: "movegpt/index.js",
```
Comment (0xngmi) [inline defi/src/protocols/data3.ts:45490]: this doesn't exist https://github.com/DefiLlama/DefiLlama-Adapters/blob/main/projects/movegpt/index.js

**Review by 0xngmi — CHANGES_REQUESTED** [review]
Don't submit a PR to this repo, instead you should submit it to https://github.com/DefiLlama/DefiLlama-Adapters

File: defi/src/protocols/data3.ts (hunk 1 above)
Comment (ken-movegpt) [reply defi/src/protocols/data3.ts:45490]: we submited https://github.com/DefiLlama/DefiLlama-Adapters/pull/10547, please help me check

Comment (ken-movegpt) [reply defi/src/protocols/data3.ts]: We are update my MGPT token, please check

Comment (ken-movegpt) [reply defi/src/protocols/data3.ts]: Please check , mate

**Review by ken-movegpt — COMMENTED** [review]
update

**Pull request closed without merging**